src/styles.ts

```diff
@@ -461,5 +462,132 @@ export const editionsPageFonts = `
 		some('italic'),
 		'assets/fonts/GHGuardianHeadline-BoldItalic.ttf',
 	)}
+`;
+
+const stage = Stage === 'CODE' ? 'code' : 'prod';
+const s3Path = `https://editions-published-${stage}.s3.eu-west-1.amazonaws.com`;
```

Comment:
can we extract this path to a commonplace? it is now defined in more than once place?